package.json

```diff
@@ -5,14 +5,16 @@
   "main": "server.js",
   "scripts": {
     "start": "node server.js",
-    "test": "newman run -e ./test/localhost3000.postman_environment.json test/PactflowProductsAPI.postman_collection.json",
+    "pre:test": "rimraf newman",
```

Comment:
If you change this to `pretest` it will automatically run when somebody runs `npm t` (see https://docs.npmjs.com/cli/v6/using-npm/scripts#pre--post-scripts)